Dockerfile

```diff
@@ -9,8 +9,9 @@ WORKDIR /app
 COPY package.json .
 COPY package-lock.json .
 RUN npm install
+RUN npm build
 COPY . .
 
 EXPOSE 80
 
-CMD ["npm", "start"]
+CMD ["npm", "start:prod"]
```

Comment:
There's no `command` block in the docker-compose file. Does this mean that during development we'll be running start:prod? is that correct for local development?

Reply:
You're right, that should be overwritten with `npm start` to enable livereload.